src/System.Management.Automation/engine/debugger/debugger.cs

```diff
@@ -4157,7 +4157,24 @@ internal void TraceVariableSet(string varName, object value)
             // because 'ToStringParser' would iterate through the enumerator to get the individual elements, which will
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
Test-Connection.TraceRoute.TraceRoute works
```powershell
Expected exactly '10.79.8.95', but got 127.0.0.1.
at <ScriptBlock>, /Users/runner/runners/2.165.0/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 257
257:             $result[0].TargetAddress | Should -BeExactly $realAddress
```